.github/workflows/release-charts.yaml

```diff
@@ -0,0 +1,118 @@
+# Workflow to tag and publish swan/ and swan-cern/ charts in this repository
+# - uses secrets.WORKFLOW_ACCESS_TOKEN (GitHub PAT with admin permissions on repostiory) to push commits and tags
+
+name: Release Charts
+run-name: Release ${{ inputs.bump }} version of all charts with changes by @${{ github.actor }}
+concurrency: release # Only run one 'release' workflow at a time
+
+permissions:
+  contents: write
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
+        description: "Bump version"
+        required: true
+        default: patch
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+
+jobs:
+  check_branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fail if not on master branch
+        if: github.ref != 'refs/heads/master'
+        run: exit 1
+  lint:
+    needs: [check_branch]
+    uses: ./.github/workflows/lint.yaml
+
+  list_changed_charts:
+    needs: [check_branch]
+    runs-on: ubuntu-latest
+    outputs:
+      swan_has_changed: ${{ steps.diff.outputs.swan_has_changed }}
+      swan-cern_has_changed: ${{ steps.diff.outputs.swan-cern_has_changed }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Fetch all tags and history to check for changes
+      - name: List charts with changes since last tag
+        id: diff
+        run: |
+          set -euxo pipefail
+          ./.github/workflows/list-changed-charts.sh >> $GITHUB_OUTPUT
+      - name: Fail if no charts have changes
```

Comment:
Do you need to fail in this case or just do nothing and show a message like: `There are no changes to be tagged, aborting.`

Reply:
The workflow will appear as passed, with a green tick in the Actions Tab, but both `tag_swan` and `tag_swan_cern` jobs would be skipped. 

This might be confusing if we are expecting a new tag to appear. Marking the workflow as failed better conveys that nothing happened.